src/main/markdown/doc/latest/polymer-tutorial/introduction.md

```diff
@@ -3,7 +3,7 @@
 ## **Disclaimer** <a id="pol-disclaimer"></a>:
 _This tutorial is based on the usage of three third-party **open-source** libraries, external to, and not managed by, the **gwtproject** organization_:
 
-* _[**Polymer**][1]: a JavaScript library for building web applications with Web Components. Polymer is owned by **Google Inc**._
+* _[**Polymer**][1]: a JavaScript library for building web applications with Web Components. Poylmer is the implementation of Google's Material Design. Polymer is owned by **Google Inc**._
```

Comment:
typo: Polymer
And  the sentence is wrong. Polymer is not the implementation of Google's Material Design. Paper elements are the polymer implementation of Material design.